### ISSUE TITLE
Xinput not installed error

### ISSUE THREAD

**Issue by greennick78** [fte-team/fteqw#248]
I am attempting to use an xbox controller for splitscreen, I am currently using joystick 1 but it does not allow me to bind the triggers. I have attempted to use in_xinput 1 but every time I do it spits out the error message "Xinput not installed". I have absolutely no clue what this means as the drivers for xinput are definitely installed on my windows 11 PC. Does anybody have any ideas?

**Comment by eukara** [comment]
Xinput is provided AFAIK by the DirectX Runtime. Is that up to date?

**Comment by Xylemon** [comment]
If you are still experiencing this issue, this is possibly related to an issue with upgrading from Windows 10 to 11:

https://answers.microsoft.com/en-us/windows/forum/all/missing-xinput13dll-for-windows-11/5bbac54c-4bca-4c50-ba47-d474e2de932e

**Comment by greennick78** [comment]
yeah, iirc the pc i was testing it on was on windows 11, ill check this out
